easybuild/easyconfigs/h/HIPS/HIPS-1.2b-rc5-foss-2017b.eb

```diff
@@ -0,0 +1,40 @@
+easyblock = 'MakeCp'
+
+name = 'HIPS'
+version = '1.2b-rc5'
+
+homepage = 'http://hips.gforge.inria.fr/'
+description = """ HIPS (Hierarchical Iterative Parallel Solver) is a scientific library that provides
+ an efficient parallel iterative solver for very large sparse linear systems. """
+
+toolchain = {'name': 'foss', 'version': '2017b'}
+toolchainopts = {'usempi': True}
+
+source_urls = ['http://hips.gforge.inria.fr/release/']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['db198f65dd4bbca25cffe537e1c659c7ca94cd5a64a32c8b9165caacb5b09603']
+
+dependencies = [
+    ('SCOTCH', '5.1.12b_esmumps'),
+]
+
+prebuildopts = 'cp Makefile_Inc_Examples/makefile.inc.gnu makefile.inc && '
+
+# make all compiles both the library and the test programs
+buildopts = (
+    'LBLAS="$LIBBLAS" '
+    'COEFTYPE=-DTYPE_REAL '
+    'PARTITIONER=-DSCOTCH_PART '
+    'ISCOTCH="-I$EBROOTSCOTCH/include" '
+    'LSCOTCH="-L$EBROOTSCOTCH/lib -lscotch -lscotcherr" '
+    'all '
+)
+
+files_to_copy = [(['LIB/*'], 'lib')]
```

Comment:
I also see `*.h` files being copied into `lib`, shouldn't we copy those to `include` instead (and adjust `sanity_check_paths` accordingly)?

```python
files_to_copy = [
    (['LIB/*.h', 'LIB/*.inc'], 'include'),
    (['LIB/*.a'], 'lib'),
]

sanity_check_paths = {
    'files': ['include/hips_fortran.h', 'include/hips.h', 'include/hips.inc', 'include/io.h', 'include/type.h',
              'lib/libhips.a', 'lib/libhipssequential.a', 'lib/libio.a', 'lib/libspkit.a'],
    'dirs': [],
}